packages/cdk-cache-cluster/lib/subnetGroup.ts

```diff
@@ -0,0 +1,23 @@
+import { Construct } from 'constructs';
+import { aws_elasticache, aws_elasticache as elasticache } from 'aws-cdk-lib';
+
+export interface SubnetGroupProps {
+    description: string,
+    subnetIds: string[],  
+    cacheSubnetGroupName: string
+}
+
+export class SubnetGroup extends Construct {
+    constructor(scope: Construct, id: string, props: SubnetGroupProps){
+        super(scope, id);
+
+        let subnetGroup: aws_elasticache.CfnSubnetGroup;
+
+        subnetGroup = new elasticache.CfnSubnetGroup(this, 'SubnetGroup', {
+            cacheSubnetGroupName: props.cacheSubnetGroupName,
+            description: props.description,
+            subnetIds: props.subnetIds
```

Comment:
Instead of the subnet ids being passed in as a property, you could make this easier to use by looking up the VPC and just having the vpc id passed in. You can use this https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-ec2.Vpc.html#static-fromwbrlookupscope-id-options. Once you have the `VPC` object, the private subnets are a property of the it.